spec/amp-cache-transform.md

```diff
@@ -0,0 +1,112 @@
+# `AMP-Cache-Transform` HTTP request header
+
+## Rationale
+
+The [Signed Exchanges](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html)
+(**SXG**) spec introduces a [new format](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#application-signed-exchange)
+for delivery of web content. AMP's [use of SXG](https://amphtml.wordpress.com/2018/01/09/improving-urls-for-amp-pages/)
+requires additional information to enable proper content negotation on a URL.
+
+### "Understands" vs "Prefers"
+
+This format should be sent in two cases:
+
+  * delivery from origin server to intermediary
+  * delivery from intermediary to user
+
+Ideally, it would *not* be sent in direct delivery from origin server to user,
+as that would best be served by a traditional HTTP exchange (e.g. requiring less
+computational overhead, and able to modify state).
+
+Therefore, the need arises for the origin to distinguish requests from users and
+requests from SXG intermediaries. That is, there is a difference between "I can
+understand the SXG format" and "I prefer an SXG if available". `Accept:
+application/signed-exchange` indicates the former. No currently-defined header
+indicates the latter.
+
+### Target-specific constraints
+
+AMP SXG are intended for [privacy-preserving
+prefetch](https://wicg.github.io/webpackage/draft-yasskin-webpackage-use-cases.html#private-prefetch)
+from a referring page (such as a Google Search results page) to a coordinating
+AMP cache (such as the Google AMP Cache). If the referrer wishes to prefetch
+subresources as well, they must also be served from a coordinating AMP cache, in
+order to preserve privacy. In order for those subresources to be *useful*, they
+must be referenced by the signed HTML page.
+
+Therefore, the requestor of an SXG may require the origin to produce an SXG
+tailored to the AMP Cache that is requesting it, by rewriting its subresource
+URLs appropriately.
+
+AMP Caches may impose additional constraints not yet specified. For instance,
+they may require the origin to apply [AMP
+transforms](amp-cache-modifications.md), and may only accept specific versions
+of those transforms.
+
+## Solution
+
+The presence of the `AMP-Cache-Transform` header indicates that the requestor
```

Comment:
Is this saying that the presence of 'AMP-Cache-Transform' alone also implies 'Accept: application/signed-exchange' and that the latter header can be omitted if the former exists?

Reply:
No, I added a sentence to clarify. Or perhaps muddy further.